internal/js/modules/k6/browser/browser/page_mapping.go

```diff
@@ -1015,7 +1015,7 @@ func waitForNavigationBodyImpl(vu moduleVU, target interface {
 		rm = newRegExMatcher(ctx, vu, tq)
 	}
 
-	return k6ext.Promise(ctx, func() (result any, reason error) {
+	return promise(vu, func() (result any, reason error) {
```

Comment:
As we don't need `ctx` here anymore, the code above could be cleaned up a bit by removing line 1003 (`ctx := vu.Context()`) and updating line 1011 with:
```
        var ctx context.Context
		ctx, stopTaskqueue = context.WithCancel(vu.Context())
```

Reply:
See #5349